### ISSUE TITLE
WebAccessibilityChecker causes problems with requirejs

### ISSUE THREAD

**Issue by ThomasZitzler** [madskristensen/WebAccessibilityChecker#9]
### Installed product versions
- Visual Studio: 2015 Enterprise Update 3
- This extension: 1.3.32
### Description

When WebAccessibilityChecker is activated, my web application is failing in the requirejs initialization (in all browsers.)

The error message shows some function with "return axe;" as error.

I would say the problem is the axe library which tried to register itself with the AMD loader in axe.min.js:
"function"==typeof define&&define.amd&&define([],function(){"use strict";return axe})
### Steps to recreate
1. Try to run any SPA application that uses requirejs
2. Start with debugger in IE, you will get the error message.

**Comment by sindilevich** [comment]
See also a bug report here: http://stackoverflow.com/q/39050189.

**Comment by Trapulo** [comment]
same problem

**Comment by mikesigs** [comment]
I am having this issue as well, quite sporadically. Another strange thing (that is somehow related) is if I have a nested require call in my HTML then the dependency I expect to be injected is replaced by an instance of axe.

``` HTML
<script src="require.js"></script>
<script>
    require(['config'], function() {
        require(['jquery'], function($) {
            $('.foo'); // => Throws exception that $ is not a function. 
        });
    });
</script>
```

If I set a breakpoint on `$('.foo')` above I see that $ is in fact the jQuery object, and no exception occurs.

**Comment by travislaynewilson** [comment]
Confirmed, I've been beating my head against my desk for hours because of this bug. Disabling the extension has fixed this for me temporarily.

**Comment by madskristensen** [comment]
Can you try the latest version 1.4 which has an updated axe-core library that doesn't seem to contain the identifier "require" at all.

**Comment by ghost** [comment]
I am still getting the issue in VS 2017 with Web Accessibility Checker 1.4.50.
Using RequireJS 2.3.3.
Looking at the BrowserLink JS under aXe v2.1.7 I can see the below code.

'define.amd && define.amd.jQuery && define("jquery",[],function(){return p})'

I am guessing that the define call may be the problem but wanted to give as much information as possible.
